worlds/minecraft/Rules.py

```diff
@@ -261,6 +264,26 @@ def get_rules_lookup(player: int):
             "Sneak 100": lambda state: can_adventure(state, player) and has_iron_ingots(state, player) and state.has("Progressive Tools", player, 2),
             "When the Squad Hops into Town": lambda state: can_adventure(state, player) and state.has("Lead", player),
             "With Our Powers Combined!": lambda state: can_adventure(state, player) and state.has("Lead", player),
+            "Smells Interesting": lambda state: can_excavate(state, player),
+            "Little Sniffs": lambda state: can_excavate(state, player),
+            "Planting the Past": lambda state: can_excavate(state, player),
+            "Crafting a New Look": lambda state: (has_iron_ingots(state, player) and # Maybe streamline this one
+                (can_adventure(state, player) or
+                    fortress_loot(state, player) or
+                    (state.can_reach("Pillager Outpost", 'Region', player) and basic_combat(state, player)) or
+                    (state.can_reach("Bastion Remnant", 'Region', player) and basic_combat(state, player)) or
+                    (state.can_reach("End City", 'Region', player) and basic_combat(state, player)))),
+            "Smithing with Style": lambda state: (can_excavate(state, player) and # Wayfinder Armor Trim
+                fortress_loot(state, player) and # Rib Armor Trim
+                (state.can_reach("Village", 'Region', player) or overworld_villager(state, player)) and  # Explorer Maps for the Vex and Tide Armor Trims
```

Comment:
I'm not sure how I feel about putting villagers as required to find these. Maybe adding compasses for the two said structures? but we already have a lot of compasses. and there is only this one advancement in these locations... so maybe ask in the MC discord channel what people think?

Reply:
Woodland Mansions are another source of Totems of Undying, so they could be an alternate requirement for Postmortal. But that's still only two Advancements, and only one for Ocean Monuments, so I think a public query would be a good idea.